test/common/protobuf/utility_test.cc

```diff
@@ -2018,4 +2018,100 @@ TEST(TypeUtilTest, TypeUrlHelperFunction) {
       TypeUtil::descriptorFullNameToTypeUrl("envoy.config.filter.http.ip_tagging.v2.IPTagging"));
 }
 
+class StructUtilTest : public ProtobufUtilityTest {
+protected:
+  ProtobufWkt::Struct updateSimpleStruct(ProtobufWkt::Value const& v0,
```

Comment:
More `const` issues. Sorry if this seems nit-picky, but this style of `const` is different to the Envoy code base and consistency is important. There are some rare exceptions and we should fix them.

Can you just grep for `const&` in your difference from `main` and fix them all? Thanks.

Reply:
I'm sorry I thought I had checked them all... I completely understand the style consistency. Will indeed recheck this properly!

Reply:
Fixed in 45ca93383ba5ad5291a6fb9bae97b4d58ebb1fb5 .

```
$ git diff main...feature/ext_add_metadata |grep 'const&' |wc -l
0
```

Hopefully I should be fine!